src/components/TextFileLink/index.android.js

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+import RNFetchBlob from 'react-native-blob-util';
+import * as FileUtils from '../../libs/fileDownload/FileUtils';
+
+const propTypes = {
+    fileName: PropTypes.string,
+    textContent: PropTypes.string,
+    children: PropTypes.func,
+};
+
+const defaultProps = {
+    fileName: '',
```

Comment:
@ArekChr I think I prefer `download` as default fileName. This way at least if something goes wrong we have a reasonable file name to the user and the app won't break. What do you think?

Reply:
Yeah, I agree!

What I'm proposing is making `fileName` required, as you suggested, AND using the string `download` as the default value, just in case somehow the types are overriden / ignored:

```js
const propTypes = {
    /** Name to the file that will be downloaded */
    fileName: PropTypes.string.isRequired,
    /** Text content to the file that will be downloaded */
    textContent: PropTypes.string.isRequired,
    children: PropTypes.func,
};

const defaultProps = {
    fileName: 'download',
    textContent: '',
    children: () => {},
};
```